app/models/hubstats/deploy.rb

```diff
@@ -1,6 +1,17 @@
 module Hubstats
   class Deploy < ActiveRecord::Base
 
+    before_validation :check_time, on: :create
+    validates :git_revision, :deployed_at, :deployed_by, :repo_id, presence: true
+    validates_associated :repo
+    validates_presence_of :repo_id
+
+    def check_time
+        if deployed_at == nil
```

Comment:
The more Ruby way to write this would be `self.deployed_at = Time.now.getutc if deployed_at.nil?`